packager/packager.go

```diff
@@ -363,6 +374,11 @@ func (this *Packager) ServeHTTP(resp http.ResponseWriter, req *http.Request, par
 			fetchResp.Header.Del(header)
 		}
 
+		// charset=utf-8 would be redundant, as it is specified in the <meta> of a valid AMPHTML document:
+		fetchResp.Header.Set("Content-Type", "text/html")
+		fetchResp.Header.Set("Content-Security-Policy", contentSecurityPolicy)
+		fetchResp.Header.Del("Link")  // Ensure there are no privacy-violating Link:rel=preload headers.
```

Comment:
Maybe also strip `Location`. Shouldn't matter if we're serving a 200, but defense in depth.

Also `Set-Cookie`. Pubs may mess this up and automatically set cookies on all cookieless requests. We don't want to hand multiple users the same identifier.

Reply:
Meh, `Location` is apparently harmless on 200, and the browser will reject signed redirects per https://wicg.github.io/webpackage/loading.html#parsing-b1 so there's no chance it'll prefetch through them. It's useless but also harmless, and I lean towards less code in general. I could be swayed.

`Set-Cookie` is stripped already, see `statefulResponseHeaders`.